odoo/http.py

```diff
@@ -1060,6 +1060,24 @@ def flatten(self):
             self.response.append(self.render())
             self.template = None
 
+    def set_cookie(self, key, value='', max_age=None, expires=None, path='/', domain=None, secure=False, httponly=False, samesite=None, cookie_type='required'):
```

Comment:
Shouldn't we go with the `def set_cookie(self, *args, **kwargs):` notation and pop the cookies type and then `super().set_cookie(*args, **kwargs)` ?